src/tracking/index.ts

```diff
@@ -125,10 +126,13 @@ export const submitProto = async (
   }, {});
   Object.entries(protoStats).forEach((stats) => {
     const [name, count] = stats;
-    statsd.increment(submissionMetricName, count as number, [
+
+    const tags = [
       `schema:${name}`,
       `status:${status}`,
-    ]);
+    ];
+
+    statsd.increment(submissionMetricName, count as number, tags);
```

Comment:
where does `submissionMetricName` coming from? I can't see it in this file, seems it will be `undefined` in runtime 🤔

Reply:
If that was the case, PR wouldn't build.  It's at the top of the file.